aikau/src/main/resources/alfresco/services/OptionsService.js

```diff
@@ -109,7 +121,22 @@ define(["dojo/_base/declare",
          {
             array.forEach(items, lang.hitch(this, this.processOptions, options, originalRequestConfig.data));
          }
-         this.alfPublish(originalRequestConfig.data.responseTopic, {
+
+         // See AKU-884...
+         // The topic that the options are currently published on is tactically chosen to only use the
+         // alfResponseScope in tandem with the alfResponseTopic - but for the responseScope to be used by itself
+         // this is to retain backwards compatibility with form controls that do not use the ServiceStore (such
+         // as select) and those that do (such as FilteringSelect)...
+         var topic;
+         if (originalRequestConfig.data.alfResponseTopic)
+         {
+            topic = (originalRequestConfig.data.alfResponseScope || "") + originalRequestConfig.data.alfResponseTopic;
```

Comment:
I'd have thought the massive comment above would have explained why this has to be done :)